test/unit/dialects/sqlite/query-generator.test.js

```diff
@@ -364,50 +364,92 @@ if (dialect === 'sqlite') {
       insertQuery: [
         {
           arguments: ['myTable', { name: 'foo' }],
-          expectation: "INSERT INTO `myTable` (`name`) VALUES ('foo');"
+          expectation: {
+            query: 'INSERT INTO `myTable` (`name`) VALUES ($1);',
+            bind: ['foo']
+          }
         }, {
           arguments: ['myTable', { name: "'bar'" }],
-          expectation: "INSERT INTO `myTable` (`name`) VALUES ('''bar''');"
+          expectation: {
+            query: 'INSERT INTO `myTable` (`name`) VALUES ($1);',
+            bind: ["'bar'"]
+          }
         }, {
           arguments: ['myTable', {data: new Buffer('Sequelize') }],
-          expectation: "INSERT INTO `myTable` (`data`) VALUES (X'53657175656c697a65');"
+          expectation: {
+            query: 'INSERT INTO `myTable` (`data`) VALUES ($1);',
+            bind: [new Buffer('Sequelize')]
+          }
         }, {
           arguments: ['myTable', { name: 'bar', value: null }],
-          expectation: "INSERT INTO `myTable` (`name`,`value`) VALUES ('bar',NULL);"
+          expectation: {
+            query: 'INSERT INTO `myTable` (`name`,`value`) VALUES ($1,$2);',
+            bind: ['bar', null]
+          }
         }, {
           arguments: ['myTable', { name: 'bar', value: undefined }],
-          expectation: "INSERT INTO `myTable` (`name`,`value`) VALUES ('bar',NULL);"
+          expectation: {
+            query: 'INSERT INTO `myTable` (`name`,`value`) VALUES ($1,$2);',
+            // TODO: Check
```

Comment:
Hmm, not sure - There is another issue about ignoring undefined completely, but it should be fine for now